docs/getting_the_most_out_of_securedrop.rst

```diff
@@ -0,0 +1,120 @@
+
+|The Intercept Become A Source|
+
+The Washington Post has a link on their front page for “how to share a tip securely”:
+
+|How to Share a Tip Securely|
+
+Other news organizations put a little link in their footer, however, we’ve found
+that this is not as effective as putting it in a more prominent on your front page.
+
+Provide Links at the Bottom of Your Articles
+--------------------------------------------
+
+Another great way to remind potential sources know that they can use SecureDrop is
+to put a link at the bottom of each article. For example, Gizmodo Media Group,
+uses a message like this:
+
+|Gizmodo Article Footer|
+
+Make an instructional video on how to access and use your SecureDrop
+--------------------------------------------------------------------
+
+To better help potential sources visualize how SecureDrop works, several
+organizations have made short instructional videos walking through all the steps.
+Some good examples include the `Toronto Globe and Mail`_, The Intercept, and
+`Lucy Parsons Labs`_.
+
+.. _`Toronto Globe and Mail`: https://www.theglobeandmail.com/news/news-video/video-a-step-by-step-guide-to-using-securedrop/article28676619/
+.. _`Lucy Parsons Labs`: https://www.youtube.com/watch?v=LkgN244ggzs
+
+Regularly share your SecureDrop landing page on social media
+------------------------------------------------------------
+
+The majority of adults in the United States now get their news from Facebook or
+other social media sites like Twitter, so it’s important to regularly remind
+people via social media posts that SecureDrop is the safest way they can contact
+your journalists if they have a sensitive tip to share. If there’s specific
+stories you are looking for tips on that may already be in the news, this is a
+great way of getting added attention to your SecureDrop.
+
+|New Yorker Tweet|
+
+Target potential whistleblowers with advertising
+------------------------------------------------
+
+Facebook and Twitter also allow for targeted advertising to users in specific
+locations, attributes, and sometimes even specific users. Gizmodo Media Group
+recently targeted online advertisements for their secure tips page at DC residents
+imploring them to `tell on trump`_. At Freedom of the Press Foundation, we ran a
+proof of concept Twitter advertisement aimed at EPA and NOAA employees to show
+how it can be done. You can read about `how you can do the same thing here`_.
+
+.. _`recently targeted online advertisements`: https://www.wsj.com/articles/gizmodo-ads-target-potential-trump-leakers-1487191482
+.. _`tell on trump`: http://tellontrump.com/
+.. _`how
+   you can do the same thing here`: https://freedom.press/news/we-targeted-securedrop-ad-potential-whistleblowers-trump-administration-you-can-too/
+
+Put an advertisement in your physical paper
+-------------------------------------------
+
+Obviously this tip only applies to news outlets that also print a physical
```

Comment:
Translation: @fowlslegs suggests "Obviously," instead of "Obviously ". I think the text is fine as-is.